resources/tests/database/database.go

```diff
@@ -4,26 +4,11 @@ import (
 	"errors"
 	"log"
 	"neurobot/infrastructure/database"
-	"path"
-	"runtime"
 
 	"github.com/upper/db/v4"
 	"github.com/upper/db/v4/adapter/sqlite"
 )
 
-func init() {
```

Comment:
Since `MakeTestDatabaseSession()` is called by each test in the testsuite, but `init()` is called only once per test suite (I think?), I think we should continue to migrate in `init()`, so that we execute migrations only once per test suite. I think this is important since run migrations make tests run slower, and as more migrations are created, the slower they will run.

Reply:
Yeah, good idea! Earlier had some trouble as closing the session on an in-memory database removes the database.

Now removed the defer call for closing the session and it worked, as there would always be an open session that's not closed. Every test opens and closes its own session.